services/blockstorage/service.go

```diff
@@ -62,12 +63,91 @@ func (s *service) CommitBlock(input *services.CommitBlockInput) (*services.Commi
 	return nil, nil
 }
 
+func (s *service) lookForTransactionsBlockHeader(height primitives.BlockHeight) (*services.GetTransactionsBlockHeaderOutput, error) {
```

Comment:
why `lookFor...`?

Reply:
because `GetTransactionsBlockHeader` and `getTransactionsBlockHeader` would be confusing

Reply:
`load`?